sentry_sdk/ai/monitoring.py

```diff
@@ -106,14 +106,14 @@ def record_token_usage(
     if ai_pipeline_name:
         span.set_attribute("ai.pipeline.name", ai_pipeline_name)
     if prompt_tokens is not None:
-        span.set_measurement("ai_prompt_tokens_used", value=prompt_tokens)
+        span.set_attribute("ai.prompt_tokens_used", prompt_tokens)
```

Comment:
[change request] Please use the attributes as documented in our [span data conventions](https://develop.sentry.dev/sdk/telemetry/traces/span-data-conventions/#ai) (note the `.` instead of `_` at the end).

```suggestion
        span.set_attribute("ai.prompt_tokens.used", prompt_tokens)
```